froide/foirequest/tests/test_web.py

```diff
@@ -534,7 +534,7 @@ def test_queries_foirequest_loggedin(self):
         - Problem reports - even for non-requester (+1)
         - ContentType + Comments for each FoiMessage (+2)
         """
-        TOTAL_EXPECTED_REQUESTS = 16
+        TOTAL_EXPECTED_REQUESTS = 18
```

Comment:
I'd really like to avoid adding queries to the request page. Especially because this looks like an 'n+1' query: an additional query per message, checking the the moderator status for every rendered comment form.

If an additional query really becomes necessary, please document it in the comment of the test.

Reply:
Yes, I was not happy with that, that's why I set this PR on draft